liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -424,7 +424,9 @@ private void logMdcData() throws IOException {
         MdcManager mdcManager = Scope.getCurrentScope().getMdcManager();
         try (MdcObject version = mdcManager.put(MdcKey.LIQUIBASE_VERSION, LiquibaseUtil.getBuildVersion());
              MdcObject systemUser = mdcManager.put(MdcKey.LIQUIBASE_SYSTEM_USER, System.getProperty("user.name"));
-             MdcObject systemName = mdcManager.put(MdcKey.LIQUIBASE_SYSTEM_NAME, NetUtil.getLocalHostName())) {
+             MdcObject systemName = mdcManager.put(MdcKey.LIQUIBASE_SYSTEM_NAME, NetUtil.getLocalHostName());
+             // The host name here is purposefully the same as the system name. The system name is retained for backwards compatibility.
+             MdcObject hostName = mdcManager.put(MdcKey.LIQUIBASE_HOST_NAME, NetUtil.getLocalHostName())) {
```

Comment:
Could you set the NetUtil.getLocalHostName() to a variable and then use it in the two setters? This method can be slow in some edge cases (it even calls getLocalHost() that can list the network interfaces to try to find the host name.)

Reply:
Good idea!